README.md

```diff
@@ -95,5 +95,5 @@ For using it with Docker, use the `--env-file .env` argument when calling `docke
 
 ## Licensing & copyright
 
-All files in this repository are [MIT licensed](LICENSE).  
+All files in this repository are [MIT licensed](LICENSE).
```

Comment:
I prefer to have a new line, but not new section here. Hence the end-of-line double spaces, which does exactly this.

Reply:
OK. Such blanks at end of line are stripped off by default by emacs...